### PR TITLE
[GitHub apps] Initial rewrite of sign up doc

### PR DESCRIPTION
**Pull request by kbatuigas**
# Description
Updates the Sign up and try section to reflect changes in the onboarding process with the GitHub App rollout.

# Reasons
At 100% rollout, all GitHub signups will go through the GitHub App onboarding flow.

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_cci2/first-steps.adoc [hunk 1]
```diff
@@ -10,62 +10,119 @@ contentTags:
 :icons: font
 :experimental:
 
-To run your very first build on CircleCI, go to the link:https://circleci.com/signup/[Sign Up] page. Sign up with your GitHub, Bitbucket or GitLab account, or your email address for the option to connect to your code later in the process.
+//TODO: Check backlinks
 
-[#vcs-signup]
-== Sign up with GitHub or Bitbucket
+To run your very first build on CircleCI, go to the link:https://circleci.com/signup/[Sign Up] page. Sign up with your email address and connect your GitHub, Bitbucket, or GitLab accounts in the process.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:15]:
```suggestion
To run your very first build on CircleCI, go to the link:https://circleci.com/signup/[Sign Up] page. Sign up with your email address and connect your GitHub, Bitbucket, or GitLab account in the process.
```

Comment (rosieyohannan) [reply jekyll/_cci2/first-steps.adoc:15]: Maybe just "To start building on CircleCI ..."?

File: jekyll/_cci2/first-steps.adoc [hunk 2]
```diff
@@ -10,62 +10,119 @@ contentTags:
...
+
+- If you are **creating a new CircleCI org** for your team, follow the xref:#sign-up-and-create-an-org[Sign up and create an org] steps below. 
+- If you are signing up for a CircleCI account through an **invite** from someone on your team, follow the steps in the xref:#sign-up-with-an-invite[Sign up with an invite] section.
+
+[#vcs-integrations]
+== VCS integrations
+
+CircleCI supports GitHub, Bitbucket, and GitLab projects. If you use **GitHub** or **GitLab**, and are setting up a CircleCI organization for your team, you will be setting the VCS integration type and authenticating with your VCS provider during the signup process. Team members that you invite afterwards to your CircleCI org will have to also later authenticate with the VCS provider if they are to create new projects.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:23]: I had to read this section a few times to work out what the difference is. Maybe the language can be simplified a bit?

File: jekyll/_cci2/first-steps.adoc [hunk 3]
```diff
@@ -10,62 +10,119 @@ contentTags:
...
+
+If you use **Bitbucket**, every team member signing up for a new account and joining your org will authenticate with Bitbucket during the signup process.
+
+[#sign-up-and-create-an-org]
+== Sign up and create an org
+
+If you are new to CircleCI, you will be creating a CircleCI organization. A CircleCI organization may correspond directly to a _personal_ CircleCI account, or more commonly, to a _shared_ account in which a team collaborates and with which multiple team members' personal CircleCI accounts can be associated. 
+
+. To create an account for free with CircleCI, enter your email address and password.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:32]: Can we add a link to the signup page in this first step

File: jekyll/_cci2/first-steps.adoc [hunk 4]
```diff
@@ -10,62 +10,119 @@ contentTags:
...
+
+. You will be taken to a screen where you can respond to prompts that best describe your role and your engineering organization.
 
-. Click on either link:https://circleci.com/auth/vcs-connect?connection=Github[**Sign Up with GitHub**] or link:https://circleci.com/auth/vcs-connect?connection=Bitbucket[**Sign Up with Bitbucket**] to start the authentication process and allow CircleCI to access your code.
+. You will be prompted to connect your code. Choose the version control system (VCS) that contains your code repositories: GitHub, GitLab.com (cloud), GitLab Self-Managed, or Bitbucket.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:36]: I think the "you will be..." sections in these steps can be cut down to leave more direct actions. Like here maybe just "Connect your code. Choose the ..."?

Comment (rosieyohannan) [reply jekyll/_cci2/first-steps.adoc:36]: Also as VCS is used above move the long version up to the first instance of this on the page?

File: jekyll/_cci2/first-steps.adoc [hunk 5]
```diff
@@ -10,62 +10,119 @@ contentTags:
...
+
+[#GitHub]
+=== GitHub
+
+. For GitHub, you will be taken to a screen to install GitHub Apps. In this step, you can specify exactly which repositories CircleCI can access, or opt to give CircleCI access to all repositories in your GitHub organization.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:41]:
```suggestion
. For GitHub, you will be taken to a screen to install the CircleCI GitHub App. In this step, you can specify exactly which repositories CircleCI can access, or opt to give CircleCI access to all repositories in your GitHub organization.
```

File: jekyll/_cci2/first-steps.adoc [hunk 6]
```diff
@@ -10,62 +10,119 @@ contentTags:
...
+
+. Next, you will be redirected to the Create New Project page where you can set up your first project and run your first pipeline in CircleCI. You will need an SSH key that will also be added as a deploy key in GitHub. Follow the instructions on the page, or refer to the xref:create-project#[Creating a Project] guide for more details.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:43]: The user will create the SSH key and also go and add the public key as a deploy key themselves for every project, maybe needs a slight rephrase here?

File: jekyll/_cci2/first-steps.adoc [hunk 7]
```diff
@@ -10,62 +10,119 @@ contentTags:
...
 +
-NOTE: If you are using GitHub, you have the option to limit CircleCI, preventing access to your private repositories. To do this, use the drop down menu at the side of the Sign Up button, and select **Public Repos Only** from the list.
+If you do not want to connect to your code at this time and only wish to continue with the email signup, click btn:[Cancel]. 
 
-. Type your GitHub or Bitbucket username, password, and two-factor authorization if applicable, then click **Sign In/Login**.
+. You will be taken to the dashboard in the CircleCI web app. From here, you can start to trigger and run project pipelines, create more projects, and invite team members to your org.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:47]:
```suggestion
. You will be taken to the dashboard in the CircleCI web app. From here, you can start to trigger pipelines, create more projects, and invite team members to your org.
```

File: jekyll/_cci2/first-steps.adoc [hunk 8]
```diff
@@ -10,62 +10,119 @@ contentTags:
...
---
+[#sign-up-with-an-invite]
+== Sign up with an invite
 
-NOTE: The full set of documentation for integrating GitLab with CircleCI can be found on the link:/docs/gitlab-integration[GitLab integration page].
+If you are new to CircleCI and signing up using an invite from a team member, you will be _joining_ a CircleCI organization. The CircleCI organization is a _shared_ account where a team collaborates and with which multiple team members' personal CircleCI accounts can be associated. 
 
-[#email-signup]
-== Sign up with email
+[NOTE]
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:90]: I think this should maybe be a CAUTION?

File: jekyll/_cci2/first-steps.adoc [hunk 9]
```diff
@@ -10,62 +10,119 @@ contentTags:
...
+NOTE: The full set of documentation for integrating Bitbucket with CircleCI can be found on the xref:bitbucket-integration#[Bitbucket integration page].
 
-The new organization allows you to manage users and GitLab projects within the organization. The organization will be under its own Free plan to start. You can upgrade to another plan later. This pricing plan is independent of any GitHub or Bitbucket organizations you have previously set up with your account.
---
+[#sign-up-with-an-invite]
+== Sign up with an invite
 
-NOTE: The full set of documentation for integrating GitLab with CircleCI can be found on the link:/docs/gitlab-integration[GitLab integration page].
+If you are new to CircleCI and signing up using an invite from a team member, you will be _joining_ a CircleCI organization. The CircleCI organization is a _shared_ account where a team collaborates and with which multiple team members' personal CircleCI accounts can be associated.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:88]:
```suggestion
If you are signing up using an invite from a team member, you will be _joining_ a CircleCI organization. The CircleCI organization is a _shared_ account where a team collaborates, and with which multiple team members' personal CircleCI accounts can be associated. 
```

File: jekyll/_cci2/first-steps.adoc (hunk 2 above)
Comment (kbatuigas) [reply jekyll/_cci2/first-steps.adoc:23]: Ah, I think I was trying to allude to who's required to do an OAuth handshake and when as described here https://circleci.atlassian.net/wiki/spaces/PES/pages/6891044888/GitHub+App+User+Flows#New-user-joining-a-GitHub-App-Org I'll think on rephrasing this!

Comment (rosieyohannan) [reply jekyll/_cci2/first-steps.adoc:23]: Yeah it does totally make sense what you have I just took a few goes to get the difference between how it works for standalone and non-standalone I guess.

File: jekyll/_cci2/first-steps.adoc [hunk 10]
```diff
@@ -10,62 +10,101 @@ contentTags:
...
+- If you are signing up for a CircleCI account through an **invite** from someone on your team, follow the steps in the xref:#sign-up-with-an-invite[Sign up with an invite] section.
 
-. Click on either link:https://circleci.com/auth/vcs-connect?connection=Github[**Sign Up with GitHub**] or link:https://circleci.com/auth/vcs-connect?connection=Bitbucket[**Sign Up with Bitbucket**] to start the authentication process and allow CircleCI to access your code.
+[#vcs-integrations]
+== VCS integrations
+
+CircleCI supports GitHub, Bitbucket, and GitLab projects. If you use **GitHub** or **GitLab**, and are creating a CircleCI organization for your team, you will set the version control system (VCS) integration type and authenticate with your VCS provider during the signup process. Team members that you invite afterwards to your CircleCI org can join in CircleCI without also authenticating with your VCS. However, if they are to create new projects, they will need to authenticate during the project creation process.
+
+If you use **Bitbucket**, every team member signing up for a new account and joining your org will authenticate with Bitbucket _during_ the signup process.
```
Comment (kbatuigas) [inline jekyll/_cci2/first-steps.adoc:23]: @rosieyohannan Thoughts on this section?

File: jekyll/_cci2/first-steps.adoc [hunk 11]
```diff
@@ -10,62 +10,101 @@ contentTags:
...
+Note that if you are setting up a project for the first time, you may need to authenticate with your VCS provider. Once you have completed a one-time authentication, you will be able to set up subsequent projects in CircleCI more quickly. Refer to the xref:create-project#[Creating a Project] guide for more information. 
 
-You will be able to start exploring features such as link:/docs/pipelines[pipelines] and link:/docs/workflows[workflows]. The **Dashboard**, **Projects**, **Organization Settings**, and **Plan** pages are not available until you connect your code.
+The full set of documentation for integrating GitHub, Bitbucket, or GitLab projects are available as follows:
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:103]:
```suggestion
Guides for integrating GitHub, Bitbucket, or GitLab projects are available as follows:
```

Comment (rosieyohannan) [reply jekyll/_cci2/first-steps.adoc:103]: just a suggestion

File: jekyll/_cci2/first-steps.adoc [hunk 12]
```diff
@@ -10,62 +10,101 @@ contentTags:
...
+If you are signing up using an invite from a team member, you will be _joining_ a CircleCI organization. The CircleCI organization is a _shared_ account where a team collaborates, and with which multiple team members' personal CircleCI accounts can be associated. 
 
-NOTE: The full set of documentation for integrating GitLab with CircleCI can be found on the link:/docs/gitlab-integration[GitLab integration page].
+[CAUTION] 
+====
+You must follow the link from an invite email in order to join an existing CircleCI organization. If you attempt to sign up directly from the link:https://circleci.com/signup/[Sign Up] page, you will be creating a new organization. 
 
-[#email-signup]
-== Sign up with email
+If you have not received an invite, please contact your org administrator.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:92]:
```suggestion
If you have not received an invite you can contact your org administrator.
```

File: jekyll/_cci2/first-steps.adoc [hunk 13]
```diff
@@ -10,62 +10,101 @@ contentTags:
...
+
+. Once you have authenticated, you will be redirected to your organization dashboard in the CircleCI web app. From here, you can start to trigger and run project pipelines, set up more projects, and invite team members to your org.
++
+For more details on setting up a project in CircleCI, read the xref:create-project#[Creating a Project] guide.
+
+** If you choose not to connect your code at this time, you can still explore some example projects within the CircleCI app. You can take a look at a popular open source project building on CircleCI (link:https://app.circleci.com/pipelines/github/facebook/react[React by Facebook]), or one of our own sample projects: a link:https://app.circleci.com/pipelines/github/CircleCI-Public/sample-javascript-cfd/[sample JavaScript app], and a link:https://app.circleci.com/pipelines/github/CircleCI-Public/sample-python-cfd/[sample Python app].
++
+When you are ready, you can connect Bitbucket account from the CircleCI web app. Click your user icon in the lower left corner to open your *User Settings*. Under *Account Integrations*, click btn:[Connect] to authenticate with Bitbucket and start setting up your projects.
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:79]:
```suggestion
When you are ready, you can connect your Bitbucket account from the CircleCI web app. Click your user icon in the lower left corner to open your *User Settings*. Under *Account Integrations*, click btn:[Connect] to authenticate with Bitbucket and start setting up your projects. 
```

File: jekyll/_cci2/first-steps.adoc [hunk 14]
```diff
@@ -10,62 +10,101 @@ contentTags:
...
-You will be able to start exploring features such as link:/docs/pipelines[pipelines] and link:/docs/workflows[workflows]. The **Dashboard**, **Projects**, **Organization Settings**, and **Plan** pages are not available until you connect your code.
+The full set of documentation for integrating GitHub, Bitbucket, or GitLab projects are available as follows:
 
-When you are ready, you can connect to your GitHub, BitBucket, or GitLab accounts from the CircleCI web app.
+- xref:github-apps-integration#[GitHub App integration]
+- xref:bitbucket-integration#[Bitbucket integration]
+- xref:gitlab-integration#[GitLab integration]
 
 [#terms]
 == Terms
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:110]: I think this section needs checking for standalone projects maybe as the permissions are so different @sfranks01 ?

File: jekyll/_cci2/first-steps.adoc [hunk 15]
```diff
@@ -10,62 +10,101 @@ contentTags:
...
+If you are new to CircleCI, you will create a CircleCI organization. A CircleCI organization may correspond directly to a _personal_ CircleCI account, or more commonly, to a _shared_ account in which a team collaborates and with which multiple team members' personal CircleCI accounts can be associated. 
+
+. To create an account for free with CircleCI, enter your email address and password on the link:https://circleci.com/signup/[Sign Up] page.
+
+. On the next screen, respond to prompts that best describe your role and your engineering organization.
+
+. Then, connect your code. Choose the VCS that contains your code repositories: GitHub, GitLab.com (cloud), GitLab Self-Managed, or Bitbucket.
+
+[#GitHub]
+=== GitHub
```
Comment (rosieyohannan) [inline jekyll/_cci2/first-steps.adoc:37]: I reckon a banner here for rollout week would be good, maybe we can say some users may have an experience more in line with Bitbucket for this time?

Comment (rosieyohannan) [reply jekyll/_cci2/first-steps.adoc:37]: With a link to the GitHub OAuth integration page

**Review by rosieyohannan — COMMENTED** [review]
This is looking really great!! Just a few suggestions

File: jekyll/_cci2/first-steps.adoc (hunk 15 above)
Comment (kbatuigas) [reply jekyll/_cci2/first-steps.adoc:37]: @rosieyohannan A banner could be good. I think the other option would be to go live with this page only at or close to 100% rollout.

I'm leaning a bit more towards the banner

**Review by rosieyohannan — APPROVED** [review]

File: jekyll/_cci2/first-steps.adoc [hunk 16]
```diff
@@ -10,62 +10,104 @@ contentTags:
...
+
+NOTE: **GitHub authentication with CircleCI is changing**. Starting August 2023 when you authenticate your CircleCI account with GitHub, your sign-up experience may reflect either the process described in this section, or the process described in the xref:#bitbucket[Bitbucket] section. For more information, read the xref:github-integration[GitHub OAuth integration page].
```
[the remark this answers is not in the document]
Comment (laureenmaCCI) [reply jekyll/_cci2/first-steps.adoc:39]: Left a comment on the google doc but for increased vis, this should say GitHub authorization with CircleCI is changing, not authentication

**Pull request merged**